### ISSUE TITLE
Timer Evaluating Runtime Error

### ISSUE THREAD

**Issue by christophe-g** [GoogleChrome/lighthouse#539]
Hi, 

Running into the following error when trying out for my test site: 

```sh
Timer Evaluating: Time To Interactive (alpha) 3790ms +0ms
  statusEnd Evaluating: Time To Interactive (alpha) +0ms
Runtime error encountered: [TypeError: Parameter 'url' must be a string, not undefined]
TypeError: Parameter 'url' must be a string, not undefined
    at Url.parse (url.js:90:11)
    at urlParse (url.js:84:5)
    at Url.resolve (url.js:426:29)
    at Object.urlResolve [as resolve] (url.js:422:40)
    at Function.audit (/usr/local/lib/node_modules/lighthouse/lighthouse-core/audits/cache-start-url.js:58:26)
    at /usr/local/lib/node_modules/lighthouse/lighthouse-core/runner.js:70:38
    at Array.map (native)
    at /usr/local/lib/node_modules/lighthouse/lighthouse-core/runner.js:67:61

```

npm@3.6.0
node@v5.6.0
DISTRIB_ID=LinuxMint
DISTRIB_RELEASE=17.2
DISTRIB_CODENAME=rafaela
DISTRIB_DESCRIPTION="Linux Mint 17.2 Rafaela"
NAME="Ubuntu"
VERSION="14.04.2 LTS, Trusty Tahr"
ID=ubuntu
ID_LIKE=debian

Cheers and thanks,
C.

**Comment by wardpeet** [comment]
can you tell us an url you're trying?

**Comment by christophe-g** [comment]
of course : 
https://test.preignition.org/
C

**Comment by wardpeet** [comment]
When running from source using this command: `npm start https://test.preignition.org` no issues.
Running `lighthouse https://test.preignition.org/`also works.
What version of lighthouse are you using? (npm is now on 1.06)

**Comment by paulirish** [comment]
Fixed by #558

**Comment by christophe-g** [comment]
Confirmed - latest commit fixes this issue, thanks,
C.
